components/footer.tsx

```diff
@@ -2,72 +2,101 @@ import React from "react";
+                <LanguageSwitcher/>
+                <Row className="d-flex justify-content-center mt-4">
+                    {/* another temp inline style maxWidth */}
+                    <div data-testid='linksList' className="d-flex flex-column px-3 gap-0 text-white" style={{maxWidth:'600px'}}>
+                        <p className="m-1 fs-5 fw-bold">{t("footer.map.title")}</p>
+                        <Link href="/topics" className="m-1 fs-6 fw-lighter text-white text-decoration-none">{t("footer.map.link1")}</Link>
+                        <Link href="/downloads" className="m-1 fs-6 fw-lighter text-white text-decoration-none">{t("footer.map.link2")}</Link>
+                        <Link href="/stories" className="m-1 fs-6 fw-lighter text-white text-decoration-none">{t("footer.map.link3")}</Link>
+                        <Link href="/purchase" className="m-1 fs-6 fw-lighter text-white text-decoration-none">{t("footer.map.link4")}</Link>
+                        <Link href="/about" className="m-1 fs-6 fw-lighter text-white text-decoration-none">{t("footer.map.link5")}</Link>
+                    </div>
+                </Row>
+                <Row className="d-flex justify-content-center align-items-center pt-3 pb-2">
+                    {/* another temp inline-style for an image */}
+                    <Image style={{maxHeight:'50px', width:'auto'}} src="/bts-crane-blue-logo-en.png" alt="logo"></Image>
+                </Row>
+                <Row className="text-center px-5">
+                    <p className="text-white fs-6 fw-light">{t("footer.missionStatement")}</p>
+                </Row>
+                <Row className="d-flex justify-content-center align-items-center pt-1 pb-3 text-white">
+                    <IconContext.Provider value={{size:'25px'}}>
+                        <BsYoutube className="footer-icon"></BsYoutube>
+                        <BsInstagram className="footer-icon"></BsInstagram>
+                        <MdEmail className="footer-icon"></MdEmail>
+                        <FaAmazon className="footer-icon"></FaAmazon>
+                    </IconContext.Provider>
+                </Row>
+                <Row className="text-center py-3 px-5">
+                    <p className="text-white fs-6 fw-light" data-testid={'copyright'}>{t("footer.copyright")}</p>
+                </Row>
+            </Container>
+        </div>
+        {/* Large screens footer starts here */}
+        {/* Test ids need to be modified to stop duplicates */}
+        <div className="w-100 d-none d-lg-flex">
+            <Container fluid className="bg-primary pt-3 d-flex" data-testid="footer-2">
+                <Col className="d-flex justify-content-center my-4" sm={4}>
+                    {/* another temp inline style maxWidth */}
+                    <div data-testid='linksList-2' className="d-flex flex-column px-3 gap-0 text-white" style={{maxWidth:'600px'}}>
+                        <p className="m-1 fs-5 fw-bold">Site Map</p>
```

Comment:
missed a footer.map.title & feels a bit inconsistent with the other files which have all the t("...") statements pulled out to a const?